spec/bump_spec.rb

```diff
@@ -332,10 +332,10 @@ module Foo
       RUBY
       `git add Gemfile #{gemspec}`
       Bundler.with_clean_env { run("bundle") }
+      `git add Gemfile.lock`
```

Comment:
before the '--no-bundle' test passed even without the `--no-bundle` option ... making the default case bundle and all the others do something to break